src/components/BenefitPackageMembersSearcher.js

```diff
@@ -0,0 +1,254 @@
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import {
+  formatMessageWithValues,
+  formatDateFromISO,
+  formatMessage,
+  Searcher,
+  downloadExport,
+  CLEARED_STATE_FILTER,
+} from '@openimis/fe-core';
+import {
+  IconButton,
+  Tooltip,
+  Dialog,
+  Button,
+  DialogActions,
+  DialogTitle,
+} from '@material-ui/core';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import HistoryIcon from '@material-ui/icons/History';
+import SentimentVeryDissatisfiedIcon from '@material-ui/icons/SentimentVeryDissatisfied';
+import { connect, useDispatch } from 'react-redux';
+import {
+  BENEFIT_PLAN_LABEL,
+  DEFAULT_PAGE_SIZE, EMPTY_STRING, MODULE_NAME, RIGHT_INDIVIDUAL_UPDATE, ROWS_PER_PAGE_OPTIONS,
+} from '../constants';
+import BenefitPackageMembersFilters from './BenefitPackageMembersFilters';
+import { applyNumberCircle } from '../util/searcher-utils';
+
+function BenefitPackageMembersSearcher({
+  rights,
+  modulesManager,
+  history,
+  intl,
+  membersPageInfo,
+  readOnly,
+  groupBeneficiaries: { group },
+  members,
+  membersTotalCount,
+  fetchedMembers,
+  fetchingMembers,
+  errorMembers,
+  membersExport,
+  errorMembersExport,
+  benefitPlan,
+}) {
+  const dispatch = useDispatch();
+
+  const fetchIndividualsRef = modulesManager.getRef('individual.actions.fetchIndividuals');
+
+  const downloadIndividualsRef = modulesManager.getRef('individual.actions.downloadIndividuals');
+
+  const openIndividual = (individual) => history.push(
+    `/${modulesManager.getRef('individual.route.individual')}`
+    + `/${individual?.id}`,
+  );
+
+  const fetch = (params) => dispatch(fetchIndividualsRef(params));
+
+  const downloadIndividuals = (params) => dispatch(downloadIndividualsRef(params));
+
+  const headers = () => {
+    const headers = [
+      'individual.firstName',
+      'individual.lastName',
+      'individual.dob',
+    ];
+    if (rights.includes(RIGHT_INDIVIDUAL_UPDATE)) {
+      headers.push('emptyLabel');
+    }
+    return headers;
+  };
+
+  const itemFormatters = () => {
+    const formatters = [
+      (individual) => individual.firstName,
+      (individual) => individual.lastName,
+      (individual) => (individual.dob ? formatDateFromISO(modulesManager, intl, individual.dob) : EMPTY_STRING),
+    ];
+    if (rights.includes(RIGHT_INDIVIDUAL_UPDATE)) {
```

Comment:
Note: both grievances and payments will be added through contribution point, but since we don't have those modules yes, we can keep this a a placeholder.